cmd/dlt/operatorrole/cmd.go

```diff
@@ -341,6 +362,30 @@ func buildCommand(r *rosa.Runtime, roleNames []string, policyMap map[string][]st
 			AddParam(awscb.RoleName, roleName).
 			Build()
 		commands = append(commands, deleteRole)
+
+		if len(hcpSharedVpcPoliciesOutput) > 0 { // Delete HCP shared VPC policies
+			for _, hcpSharedVpcPolicy := range hcpSharedVpcPoliciesOutput {
+				hasRhManagedTag := false
+				hasHcpSharedVpcTag := false
+				for _, tag := range hcpSharedVpcPolicy.Policy.Tags {
+					if *tag.Key == tags.RedHatManaged {
+						hasRhManagedTag = true
+					} else if *tag.Key == tags.HcpSharedVpc {
+						hasHcpSharedVpcTag = true
+					}
+				}
+				if hasHcpSharedVpcTag && hasRhManagedTag {
```

Comment:
is it possible that a policy only has tag tag.HcpSharedVpc, but doesn't have tags.RedHatManaged?

Reply:
hm, I guess not no, maybe its not needed, but the policies created will always have both tags, WDYT?

Reply:
maybe it's not needed to check for both* but it's not possible to create these policies without both